### PR TITLE
init ARGC 

### PR DESCRIPTION
**Pull request by slowcoders**
It is Garbage collection enabled J2ObjC version.

### CODE REVIEW

File: jre_emul/android/platform/aosp/src/main/java/javax/net/ssl/X509ExtendedTrustManager.java [hunk 1]
```diff
@@ -0,0 +1,234 @@
+/*
```
Comment (tomball) [inline jre_emul/android/platform/aosp/src/main/java/javax/net/ssl/X509ExtendedTrustManager.java:1]: Where is the repository for this package's sources? https://android.googlesource.com/ doesn't have a platform/aosp repository.

File: make/static_framework.mk [hunk 2]
```diff
@@ -117,12 +117,12 @@ $(FRAMEWORK_HEADER):
...
-	@clang -c -o $(FRAMEWORK_HEADER:%.h=%.o) $(VERIFY_FLAGS) -x objective-c -std=c11 -fno-objc-arc $@
-	@clang -c -o $(FRAMEWORK_HEADER:%.h=%.o) $(VERIFY_FLAGS) -x objective-c++ -std=c++11 $@
-	@clang -c -o $(FRAMEWORK_HEADER:%.h=%.o) $(VERIFY_FLAGS) -x objective-c++ -std=c++11 -fobjc-arc $@
-	@clang -c -o $(FRAMEWORK_HEADER:%.h=%.o) $(VERIFY_FLAGS) -x objective-c++ -std=c++11 \
+	@clang -c -o $(FRAMEWORK_HEADER:%.h=%.o) $(VERIFY_FLAGS) -x objective-c -std=gnu11 $@
+	@clang -c -o $(FRAMEWORK_HEADER:%.h=%.o) $(VERIFY_FLAGS) -x objective-c -std=gnu11 -fobjc-arc $@
+	@clang -c -o $(FRAMEWORK_HEADER:%.h=%.o) $(VERIFY_FLAGS) -x objective-c -std=gnu11 -fno-objc-arc $@
+	@clang -c -o $(FRAMEWORK_HEADER:%.h=%.o) $(VERIFY_FLAGS) -x objective-c++ -std=gnu++14 $@
+	@clang -c -o $(FRAMEWORK_HEADER:%.h=%.o) $(VERIFY_FLAGS) -x objective-c++ -std=gnu++14 -fobjc-arc $@
+	@clang -c -o $(FRAMEWORK_HEADER:%.h=%.o) $(VERIFY_FLAGS) -x objective-c++ -std=gnu++14 \
```
Comment (tomball) [inline make/static_framework.mk:125]: Why is gnu11 and gnu++14 needed?

File: jre_emul/android/platform/aosp/src/main/java/javax/net/ssl/X509ExtendedTrustManager.java (hunk 1 above)
Comment (tomball) [reply jre_emul/android/platform/aosp/src/main/java/javax/net/ssl/X509ExtendedTrustManager.java]: Thanks for deleting these sources.

File: make/static_framework.mk (hunk 2 above)
Comment (slowcoders) [reply make/static_framework.mk:125]: Sorry. I can't remember why I changed that. -.-;;
I have rollbacked the compile options.

**Pull request merged**